product/views.py

```diff
@@ -1,3 +1,90 @@
-from django.shortcuts import render
+from django.views   import View
+from django.http    import JsonResponse
 
-# Create your views here.
+from .models        import MainCategory, Product, ProductGroup
+
+class CategoryView(View):
```

Comment:
카테고리 리스트를 반환해주는 엔드포인트이므로, class명을 Categories로 만들어주시는게 좋을 것 같습니다.
```suggestion
class CategoriesView(View):
```